data/gui/themes/default/widget/window_transparent.cfg

```diff
@@ -0,0 +1,93 @@
+#textdomain wesnoth-lib
+###
+### Definition of transparent dialogs.
+###
+
+#define _GUI_RESOLUTION_TRANSPARENT_BASE RESOLUTION BORDER_SIDE_1 BORDER_SIDE_2 _BACKGROUND_DRAW
+	[resolution]
+
+		{RESOLUTION}
+
+		left_border = {BORDER_SIDE_1}
+		right_border = {BORDER_SIDE_2}
+		top_border = {BORDER_SIDE_1}
+		bottom_border = {BORDER_SIDE_2}
+
+		[background]
+
+			[draw]
+
+				{_BACKGROUND_DRAW}
+
+			[/draw]
+
+		[/background]
+
+		[foreground]
+
+			[draw]
+			[/draw]
+
+		[/foreground]
+
+		[grid]
+
+			[row]
+				grow_factor = 1
+				[column]
+					horizontal_grow = true
+					vertical_grow = true
+					[scrollbar_panel]
+						vertical_scrollbar_mode = "initial_auto"
+						horizontal_scrollbar_mode = "initial_auto"
+						[definition]
+
+							[row]
+
+								[column]
+									horizontal_grow = true
+									vertical_grow = true
+									[grid]
+										id = "_window_content_grid"
+									[/grid]
+
+								[/column]
+
+							[/row]
+
+						[/definition]
+
+					[/scrollbar_panel]
+
+				[/column]
+
+			[/row]
+
+			[row]
+
+				[column]
+
+					[button]
+						id = "click_dismiss"
+						definition = "default"
+
+						label = _ "Close"
+					[/button]
```

Comment:
This button is probably not needed for the design you are specifying. The person using this window definition can add their own close buttons if needed, unless removing it causes a problem.

Reply:
This button also can be found in window_default.cfg and window_borderless.cfg. It (as well as scrollbar) will appear only when there is not enough space on the screen to display content. And so, if dialog is well designed, these things won't appear at all